examples/10_stormcast_ensemble_example.py

```diff
@@ -0,0 +1,276 @@
+#    :language: python
+#    :start-after: # sphinx - deterministic start
+#    :end-before: # sphinx - deterministic end
+
+# %%
+# Thus, we need the following:
+#
+# - Prognostic Model: Use the built in StormCast Model :py:class:`earth2studio.models.px.StormCast`.
+# - perturbation_method: Use the Zero Method :py:class:`earth2studio.perturbation.Zero`. We will not
+#    perturb the initial data because StormCast has stochastic generation of  ensemble members.
+# - Datasource: Pull data from the HRRR data api :py:class:`earth2studio.data.HRRR`.
+# - IO Backend: Let's save the outputs into a Zarr store :py:class:`earth2studio.io.ZarrBackend`.
+#
+# StormCast also requires a conditioning data source. We use a forecast data source here,
+# GFS_FX :py:class:`earth2studio.data.GFS_FX`, but a non-forecast data source such as ARCO 
+# could also be used with appropriate time stamps.
+
+# %%
+import os
+
+os.makedirs("outputs", exist_ok=True)
+from dotenv import load_dotenv
+
+load_dotenv()  # TODO: make common example prep function
+
+from earth2studio.data import ARCO, HRRR
+from earth2studio.io import ZarrBackend
+from earth2studio.models.px import StormCast
+from earth2studio.perturbation import Zero
+
+
+# Load the default model package which downloads the check point from NGC
+package = StormCast.load_default_package()
+model = StormCast.load_model(package)
+
+# Instantiate the (Zero) perturbation method 
+z = Zero()
+
+# Create the data source
+data = HRRR()
+
+# Create and set the conditioning data source
+conditioning_data_source = ARCO()
+model.conditioning_data_source = conditioning_data_source
+
+# Create the IO handler, store in memory
+io = ZarrBackend()
+
+# %%
+# Execute the Workflow
+# --------------------
+# With all components initialized, running the workflow is a single line of Python code.
+# Workflow will return the provided IO object back to the user, which can be used to
+# then post process. Some have additional APIs that can be handy for post-processing or
+# saving to file. Check the API docs for more information.
+#
+# For the forecast we will predict for 4 hours 
+
+# %%
+import earth2studio.run as run
+
+nsteps = 4
+nensemble = 8
+batch_size = 2 
+
+
+date = "2022-11-04T21:00:00"
+io = run.ensemble(
+    [date], 
+    nsteps, 
+    nensemble,
+    model, 
+    data, 
+    io,
+    z,
+    batch_size = batch_size,
+    output_coords = {"variable": np.array(["t2m", "refc"])}
+)
+
+print(io.root.tree())
+
+# %%
+# Post Processing
+# ---------------
+# The last step is to post process our results. Cartopy is a great library for plotting
+# fields on projections of a sphere. Here we will just plot the temperature at 2 meters
+# (t2m) 4 hours into the forecast.
+#
+# Notice that the Zarr IO function has additional APIs to interact with the stored data.
+
+# %%
+import cartopy
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+forecast = f"{date}"
+step = 4  # lead time = 1 hr
```

Comment:
For outputs/plotting, the lead time would be 4hrs if `step = 4`. Maybe just remove the lead time comment

Reply:
Fixed.